timemachine/cpp/src/context.cu

```diff
@@ -147,54 +149,44 @@ std::array<std::vector<double>, 3> Context::multiple_steps_U(
         throw std::runtime_error("store_x_interval <= 0");
     }
 
+    if (n_steps % store_x_interval != 0) {
+        std::cout << "warning:: n_steps modulo store_x_interval does not equal zero" << std::endl;
+    }
+
     int n_windows = lambda_windows.size();
-    int u_traj_size = ((n_steps + store_u_interval - 1) / store_u_interval) * n_windows;
-    int x_traj_size = (n_steps + store_x_interval - 1) / store_x_interval;
+    int u_traj_size = (n_steps / store_u_interval) * n_windows;
+    int x_traj_size = (n_steps / store_x_interval);
```

Comment:
nit: redundant parens

Reply:
Resolved in [e7ab5f5](https://github.com/proteneer/timemachine/pull/724/commits/e7ab5f54c1e718de58d841e800e2228272127329)